pkg/kubelet/kubelet.go

```diff
@@ -1668,11 +1753,18 @@ func (kl *Kubelet) tryUpdateNodeStatus() error {
 	// cAdvisor locally, e.g. for test-cmd.sh, and in integration test.
 	info, err := kl.GetCachedMachineInfo()
 	if err != nil {
-		glog.Errorf("error getting machine info: %v", err)
+		// TODO(roberthbailey): This is required for test-cmd.sh to pass.
```

Comment:
@vishh @vmarmol Can you help answer the question I put in this TODO? Should I put a default capacity here (which is what test-cmd.sh expects) or should we not be picking a random capacity if cadvisor doesn't let us know what the node capacity is (e.g. update the test to stop checking for this)? It would be better if cadvisor just worked in test-cmd.sh but until then we need to decide which way to go here. Thanks.